django_simple_api/decorators.py

```diff
@@ -1,6 +1,7 @@
 import sys
+
```

Comment:
When importing the standard library, there should be no blank line in the middle.